tests/utils.js

```diff
@@ -18,7 +18,11 @@ define([
     var assert = chai.assert,
         savedForm = null,
         saveCount = 0;
-    
+
+    // monkey-patch chai to use ===/!== instead of ==/!= in assert.equal
+    assert.equal = assert.strictEqual;
+    assert.notEqual = assert.notStrictEqual;
```

Comment:
I did this because *JavaScript the Good Parts* recommends to never use `==` or `!=`. Why would we want to use them in our tests? IMO Chai should do this by default.